cmd/ci-operator/main_test.go

```diff
@@ -412,3 +414,138 @@ func TestGetResolverInfo(t *testing.T) {
 		}
 	}
 }
+
+func TestGetConsoleURL(t *testing.T) {
+	testCases := []struct {
+		name          string
+		cm            *coreapi.ConfigMap
+		expected      string
+		expectedError error
+	}{
+		{
+			name:          "nil cm",
+			expectedError: fmt.Errorf("nil config map"),
+		},
+		{
+			name: "no data in cm",
+			cm: &coreapi.ConfigMap{
+				TypeMeta: meta.TypeMeta{},
+				ObjectMeta: meta.ObjectMeta{
+					Name:      "console-config",
+					Namespace: "openshift-console",
+				},
```

Comment:
This looks like an irrelevant boilerplate; the tested method does not care about these fields, so why set them?

Reply:
Removed.